doc: remove obsolete content in document (#7140)
projectacrn/acrn-hypervisor: doc/tutorials/realtime_performance_tuning.rst

> Enable Performance Monitoring Unit (PMU) Support in VM

Reviewer: I think we can update this title too (since PMU support is now already enabled:
```suggestion
Performance Monitoring Unit (PMU) Support for the RTVM
```

Author: updated

======================================================

> By default, the ACRN hypervisor has exposed the PMU-related CPUID and MSRs to the RTVM.

Reviewer: Grammar fix:
```suggestion
By default, the ACRN hypervisor exposes the PMU-related CPUID and MSRs to the RTVM.
```

Author: Updated

      +
      + enable_msr_interception(msr_bitmap, MSR_IA32_A_PMC0, INTERCEPT_DISABLE);
      + enable_msr_interception(msr_bitmap, MSR_IA32_A_PMC1, INTERCEPT_DISABLE);
      + enable_msr_interception(msr_bitmap, MSR_IA32_A_PMC2, INTERCEPT_DISABLE);
      + enable_msr_interception(msr_bitmap, MSR_IA32_A_PMC3, INTERCEPT_DISABLE);
      + enable_msr_interception(msr_bitmap, MSR_IA32_A_PMC4, INTERCEPT_DISABLE);
      + enable_msr_interception(msr_bitmap, MSR_IA32_A_PMC5, INTERCEPT_DISABLE);
      + enable_msr_interception(msr_bitmap, MSR_IA32_A_PMC6, INTERCEPT_DISABLE);
      + enable_msr_interception(msr_bitmap, MSR_IA32_A_PMC7, INTERCEPT_DISABLE);
      + enable_msr_interception(msr_bitmap, MSR_IA32_PERFEVTSEL0, INTERCEPT_DISABLE);
      + enable_msr_interception(msr_bitmap, MSR_IA32_PERFEVTSEL1, INTERCEPT_DISABLE);
      + enable_msr_interception(msr_bitmap, MSR_IA32_PERFEVTSEL2, INTERCEPT_DISABLE);
      + enable_msr_interception(msr_bitmap, MSR_IA32_PERFEVTSEL3, INTERCEPT_DISABLE);
      +
      /* Setup MSR bitmap - Intel SDM Vol3 24.6.9 */
      value64 = hva2hpa(vcpu->arch.msr_bitmap);
      exec_vmwrite64(VMX_MSR_BITMAP_FULL, value64);

> 

Reviewer 2: Line 155/86 below says, "After exposing PMU-related CPUID/MSRs to the VM, performance analysis tools..."

Since users no longer need to expose PMU-related CPUID/MSRs to the VM, consider changing the wording to something like, "While PMU-related CPUID/MSRs are exposed to the VM, performance analysis tools..."

Author: updated
